client/src/component/CircleChart.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from "react";
+import { PieChart, Pie, Cell, Label } from "recharts";
+
+const data = [
+  { name: "Group A", value: 400 },
+  { name: "Group B", value: 300 },
+];
+
+export const CircleChart = () => {
```

Comment:
You will want to pass in data as props to allow this to be used

Reply:
code edited